### DISCUSSION TITLE
Authentication problems

### DISCUSSION THREAD

**Discussion by dellycowboy** [mivano/azure-cost-cli#65] [Q&A]
I am struggling to authenticate, how is the code managing the oauth flow?

I can call the Billing API ok via postman with a bearer token generated with (Get-AzAccessToken).Token

When I install azure-cost-cli and try to run it via Powershell I get

Error: Response status code does not indicate success: 403 (Forbidden).

**Comment by mivano** [comment]
Make sure first to log in using the azure cli: `az login` 
If that does not help, make sure to force it to the tenant you want to access using the `--tenant` parameter. 
The app will pick up the azure cli context.
